src/Constants.scss

```diff
@@ -0,0 +1,18 @@
+﻿/* Site colours */
+$blue: rgba(36, 30, 78, 1);
+$red: rgba(150, 2, 0, 1);
+$rust: rgba(206, 108, 71, 1);
+$orange: rgba(244, 162, 97, 1);
+$green: rgba(234, 218, 162, 1);
+
+/* Site Fonts */
+$fontFamilyOpenSans:'Open Sans', sans-serif;
+$fontFamilyUbuntu:'Ubuntu', sans-serif;
+
+$paraFont-size:13.5pt;
+$quoteFont-size:.95em;
+$labelFont-size:9pt;
+$headerThreeFont-size: 1.5em;
+$headerTwoFont-size: 2em;
+$headerOneFont-size: 2.5em;
```

Comment:
These are really nice - could we edit it though to have consistent naming.

ie either always use `camelCase` or always use `things-with-dashes`.
At the moment we seem to have both kicking about.